salalib/isovist.cpp

```diff
@@ -69,51 +69,55 @@ void Isovist::makeit(BSPNode *root, const Point2f& p, const QtRegion& region, do
    m_occlusion_points.clear();
 
    bool markedcentre = false;
+   auto prev = m_blocks.begin();
+   auto curr = m_blocks.begin();
    for (size_t i = 0; i < m_blocks.size(); i++) {

```

Comment:
How about writing this as `for (;curr != m_block.end(); ++curr){` and then at the end just `prev = cur;` instead of the faffing around with `if ( i != 0 )` and so on?